dist/images/ovnkube.sh

```diff
@@ -180,6 +184,31 @@ routable_mtu=${OVN_ROUTABLE_MTU:-}
 # set metrics endpoint bind to K8S_NODE_IP.
 metrics_endpoint_ip=${K8S_NODE_IP:-0.0.0.0}
 metrics_endpoint_ip=$(bracketify $metrics_endpoint_ip)
+
+# set metrics endpoint master port bind to K8S_NODE_IP
+metrics_master_port=${OVN_METRICS_MASTER_PORT}
```

Comment:
@naserdean can we use the pattern just above here instead?

`metrics_master_port=${OVN_METRICS_MASTER_PORT:-9409}`

and same for the rest of them. Single line instead of 4 for each one then